project-root/Assets/Scripts/NavMeshSpawner.cs

```diff
@@ -39,47 +53,34 @@ bool RandomPoint(Vector3 center, float range, out Vector3 result)
     }
 
 
-     void Spawn()
+    public void Spawn()

```

Comment:
This whole method is messy. Your brackets are all out of alignment which makes this practically unreadable. Try Ctrl+K, Ctrl+D. This should help clean up any type of mess related to this. Remember, we have a style guide so follow it!